Sources/SwiftKuery/Column.swift

```diff
@@ -100,28 +145,41 @@ public class Column: Field, IndexColumn {
         }
         return result
     }
-
-    /// Build the index column using `QueryBuilder`.
-    ///
+    
+    /**
+     Function to build a String representation of the index of a `Column` instance.
+     A `QueryBuilder` is used handle variances between the various database engines and produce a correct SQL description.
+     ### Usage Example: ###
+     In this example, `QueryBuilder` and a `PersonTable` (as defined at the top of this class) instances are initialized.
+     The build function is then called to produce a String representation of the index column and the results are printed.
+     ```swift
+     let queryBuilder = QueryBuilder()
+     let personTable = PersonTable()  // PersonTable() is a previously defined `Table` class
+     let description = try personTable.name.build(queryBuilder: queryBuilder)
```

Comment:
Remove the `try` from here as this function doesn't throw.
Also the function name is `buildIndex` not `build`.